resources/public/api/conf/2.0/common/pathParameters.yaml

```diff
@@ -30,3 +30,16 @@ components:
         type: string
         example: X92AM39DLE3D
       required: true
+    chargeReference:
+      name: chargeReference
+      in: path
+      description: |
+        This field will be used when the need is to view a specific document with help of Charge Reference number 
```

Comment:
Has this description been reviewed?

- It's not a field.
- 'This field will be used when the need is to...' seems odd.
- Is the charge reference a number?

Also should there be information about where this reference be obtained (the spec has a comment about a documentID (??) field in Retrieve Self Assessment Balance and Transactions).